changelogs/fragments/954-linting.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
```

Comment:
It may be nice to isolate this specific change in a different PR.

Reply:
I'll strip it from the backports